README.md

```diff
@@ -22,7 +22,7 @@ Requirements:
   When you have a version of the GNU sed installed on a macOS you may specify the binary using
   ```bash
   # brew install gnu-sed
-  make install -e SED=/usr/local/bin/gsed
+  NAMESPACE=default make run -e SED=/usr/local/bin/gsed
```

Comment:
why should this "make install" eventually become "make run" ?

Reply:
Make install does not work at the moment, so people following the readme with get an error. Is changing it to make run bad here?

Reply:
I would say that previous instructions were wrong, as `make install` just installed CRDs, however it didn't start the operator itself.
`make run` will start the operator now, connecting to the cluster from Kubeconfig.

Reply:
@Ygnas Can you check if `-e SED=/usr/local/bin/gsed` is still needed. AFAIK there is no string replacement being done as part of `make run`

Reply:
@sutaakar Yes `make run` does not need sed to run and it works fine. So would leaving only a note in this place fine?

Reply:
if you want use "make run" here, just remove the "sed" part

Btw: @Fiona-Waters  + @dimakis looks like this PR: 
 https://github.com/project-codeflare/codeflare-operator/commit/4b863f79d4b704c511ea404e27e3ec65d66b5a0f removed the support of gsed on MacOS, was that a mistake?   FYI @tedhtchang

Reply:
@Ygnas thinking, is SED used in any other command? If not then you can remove the note too.

Reply:
ok, so just remove the SED from command

Reply:
> if you want use "make run" here, just remove the "sed" part
> 
> Btw: @Fiona-Waters + @dimakis looks like this PR: [4b863f7](https://github.com/project-codeflare/codeflare-operator/commit/4b863f79d4b704c511ea404e27e3ec65d66b5a0f) removed the support of gsed on MacOS, was that a mistake? FYI @tedhtchang

@zdtsw This was completed quickly just before a release as far as I remember. It was updated very soon afterwards in this [commit](https://github.com/project-codeflare/codeflare-operator/commit/c7954a9bd6b19f5978bcf318ebb7d839ad4b1b99) - a lot of this has been removed now with the removal of mcad. Hope I am understanding correctly.